pkg/auth/provider_oidc.go

```diff
@@ -2,29 +2,60 @@ package auth
 
 import (
 	"context"
+	"fmt"
 	"log/slog"
+	"strings"
 	"time"
 
 	"github.com/coreos/go-oidc/v3/oidc"
 )
 
 type OidcProvider struct {
 	logger           *slog.Logger
-	issuer           string
+	Issuer           string
```

Comment:
Was this change intentional? The case doesn't match the other fields in the struct

Reply:
In go upper case first letter means it can be exported and accessible by other packages. Lower case first letter is for private use. The issuer need to be accessed from auth.go file. I can update it to lower case and create a "GetIssuer" func to make it cleaner